parsers/CA_SK.py

```diff
@@ -1,142 +1,108 @@
+import typing
 from datetime import datetime, timedelta
 from logging import Logger, getLogger
 
 from pytz import timezone
```

Comment:
Since we updated python we can use the zoneinfo built-in instead, which means less external dependencies (which makes my job easier 😅)

It will also fix some odd extra minutes we have been observing with pytz for some timezones.